src/extension.ts

```diff
@@ -490,6 +490,7 @@ function launchMetals(
         ServerCommands.AmmoniteStop,
         ServerCommands.ScalaCliStart,
         ServerCommands.ScalaCliStop,
+        "open-new-github-issue",
```

Comment:
languageclient library is not really used anymore by anything aside from the Metals extension, so probably it doesn't make much sense to add it there.

We should instead migrate most of the code here, but I would like to include this command before the release.

Reply:
We can embed that library in metals-vscode but in separate dir, so in the future it might be extracted easily, if needed. But having it locally would make things easier 👍